server/src/main/kotlin/org/itmodreamteam/myrest/server/service/restaurant/RestaurantService.kt

```diff
@@ -0,0 +1,15 @@
+package org.itmodreamteam.myrest.server.service.restaurant
+
+import org.itmodreamteam.myrest.server.model.restaurant.Restaurant
+import org.itmodreamteam.myrest.shared.restaurant.RestaurantRegistrationInfo
+import org.itmodreamteam.myrest.shared.restaurant.RestaurantUpdateInfo
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.Pageable
+
+interface RestaurantService {
+
+    fun register(newRestaurant: RestaurantRegistrationInfo)
```

Comment:
Here it would be convenient to return RestaurantInfo, containing restaurant ID. Otherwise the client won't know the ID  and won't be able to display newly created Restaurant info to the user.